docs/test_frameworks/xunit_net.md

```diff
@@ -75,6 +81,8 @@ d29a393816ce   redis:7.0                   "docker-entrypoint.s…"   3 seconds
 e878f0b8f4bc   testcontainers/ryuk:0.9.0   "/bin/ryuk"              3 seconds ago
 ```
 
+To use a single container-fixture throughout all tests of a xUnit v3 test-assembly, you can use `[assembly: AssemblyFixture(typeof(YourContainerFixture))]`. The instance can then be used as a constructor-parameter like above, or be accessed through `TestContext.Current.GetFixture` in all tests of said test-assembly.
```

Comment:
```suggestion
To use a single container fixture across all tests in xUnit.net, add the `[assembly: AssemblyFixture(typeof(TContainerFixtureEntity))]` annotation to your fixture class. The fixture can then be injected into test class constructors (as shown above), or accessed using the `TestContext.Current.GetFixture<TContainerFixtureEntity>()` method (v3 only) in any test.
```

Reply:
`all tests in xUnit.net` sounds wrong to me, as it is your assembly building on top of xUnit.net. This also does feel related to how we decide in https://github.com/testcontainers/testcontainers-dotnet/pull/1442/files#r2070470394

`annotation to your fixture class` is ok with me, but not a 100% accurate. It is an assembly-annoataion refering to the fixture.